### ISSUE TITLE
BW -> color photos details

### ISSUE THREAD

**Issue by lambdaofgod** [phillipi/pix2pix#145]
In the paper there is a mention of the colorizing model. What are the details?
Specifically I'm interested in:

* training hyperparameters, especially learning rates, lambda_1 and learning rate decay
* what were the sizes of kernels in generator and discriminator

**Comment by junyanz** [comment]
You can find more training details in the appendix of the [paper](https://arxiv.org/pdf/1611.07004.pdf). 
![image](https://user-images.githubusercontent.com/1924757/42198319-49f2d81e-7e55-11e8-8fae-b6c7c11c83c8.png)

We use the same G (U-Net) and D (PatchGAN). I think better results can be obtained with application tailored networks.
